modules/arm_plugin/armplg_build.sh

```diff
@@ -0,0 +1,212 @@
+            echo "Source reloading requested"
+            echo "Removing existing sources..."
+            rm -rf $1 || fail 1 "Failed to remove. Stopping"
+            echo "Cloning master branch only..."
+            git lfs clone --recurse-submodules --single-branch --branch=master $2 $1 || fail 3 "Failed to clone $TEST_DIR_NAME. Stopping"
+        elif [ -d $1/build ]; then
+            echo "Build directory detected at $1"
+            if [ "$UPDATE_SOURCES" = "clean" ]; then
+                echo "Cleanup of previous build requested"
+                echo "Removing previous build results..."
+                rm -rf $2/build || fail 2 "Failed to cleanup. Stopping"
+            fi
+        fi
+    fi
+    return 0
+}
+
+
+
+#Prepare sources
+checkSrcTree $OPENCV_HOME https://github.com/opencv/opencv.git OpenCV
+checkSrcTree $OPENVINO_HOME https://github.com/openvinotoolkit/openvino.git OpenVINO
+checkSrcTree $OPENVINO_CONTRIB https://github.com/openvinotoolkit/openvino_contrib.git OpenVINO_contrib
+
+#cleanup package destination folder
+[ -e $STAGING_DIR ] && rm -rf $STAGING_DIR
+mkdir -p $STAGING_DIR
+
+#Build OpenCV
+mkdir -p $OPENCV_HOME/build && \
+cd $OPENCV_HOME/build && \
+PYTHONVER=`ls /usr/include | grep "python3[^m]*$"` && \
+cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_LIST=imgcodecs,videoio,highgui,python3 \
+      -DBUILD_opencv_python2=OFF -DBUILD_opencv_python3=ON -DOPENCV_SKIP_PYTHON_LOADER=ON \
+      -DPYTHON3_INCLUDE_PATH=/usr/include/${PYTHONVER}m \
+      -DPYTHON3_LIBRARIES=/usr/lib/$ARCH_NAME/lib${PYTHONVER}m.so \
+      -DPYTHON3_NUMPY_INCLUDE_DIRS=/usr/lib/python3/dist-packages/numpy/core/include \
+      -D CMAKE_USE_RELATIVE_PATHS=ON \
+      -D CMAKE_SKIP_INSTALL_RPATH=ON \
+      -D OPENCV_SKIP_PKGCONFIG_GENERATION=ON \
+      -D OPENCV_SKIP_PYTHON_LOADER=ON \
+      -D OPENCV_SKIP_CMAKE_ROOT_CONFIG=ON \
+      -D OPENCV_BIN_INSTALL_PATH=bin \
+      -D OPENCV_INCLUDE_INSTALL_PATH=include \
+      -D OPENCV_LIB_INSTALL_PATH=lib \
+      -D OPENCV_CONFIG_INSTALL_PATH=cmake \
+      -D OPENCV_3P_LIB_INSTALL_PATH=3rdparty \
+      -D OPENCV_SAMPLES_SRC_INSTALL_PATH=samples \
+      -D OPENCV_DOC_INSTALL_PATH=doc \
+      -D OPENCV_OTHER_INSTALL_PATH=etc \
+      -D OPENCV_LICENSES_INSTALL_PATH=etc/licenses \
+      -D ENABLE_CXX11=ON \
+      -D CMAKE_INSTALL_PREFIX=install \
+      -DCMAKE_TOOLCHAIN_FILE="$OPENVINO_HOME/cmake/$TOOLCHAIN_DEFS" \
+      -DCMAKE_STAGING_PREFIX=$STAGING_DIR/opencv \
+      $OPENCV_HOME && \
+make -j$BUILD_JOBS && \
+make install && \
+echo export OpenCV_DIR=\$INSTALLDIR/opencv/cmake > $STAGING_DIR/opencv/setupvars.sh && \
+echo export LD_LIBRARY_PATH=\$INSTALLDIR/opencv/lib:\$LD_LIBRARY_PATH >> $STAGING_DIR/opencv/setupvars.sh && \
+cd $DEV_HOME || fail 11 "OpenCV build failed. Stopping"
+
+#Build OpenVINO
+mkdir -p $OPENVINO_HOME/build && \
+cd $OPENVINO_HOME/build && \
+cmake -DOpenCV_DIR=$STAGING_DIR/opencv/cmake -DENABLE_OPENCV=OFF \
```

Comment:
Why `-DENABLE_OPENCV` is `OFF`?

Reply:
It's an OpenVINO legacy. Actually the flag should be named something like DOWNLOAD_PREBUILD_OPENCV